### PR TITLE
refactor: Optimise icons

### PR DESCRIPTION
**Pull request by jletey**
@ericfennis Quite a lot of cleanup here 😅

### CODE REVIEW

File: icons/contrast.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="12" cy="12" r="10"/>
-  <path d="M12 18a6 6 0 000-12v12z"/>
+  <circle cx="12" cy="12" r="10" />
+  <path d="M12 18a6 6 0 0-12 NaN NaNz" />
```
Comment (jletey) [inline icons/contrast.svg:13]: 🤔 For some reason this isn't showing up

Comment (ericfennis) [reply icons/contrast.svg:13]: maybe the path wasn't "completed"

File: icons/battery-charging.svg [hunk 2]
```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M7 7H3.77778C2.79594 7 2 7.74619 2 8.66667V15.3333C2 16.2538 2.79594 17 3.77778 17H6M14 7H16.2222C17.2041 7 18 7.74619 18 8.66667V15.3333C18 16.2538 17.2041 17 16.2222 17H13" />
+  <path d="M7 7H3.778C2.796 7 2 7.746 2 8.667v6.666C2 16.253 2.796 17 3.778 17H6m8-10h2.222C17.204 7 18 7.746 18 8.667v6.666c0 .92-.796 1.667-1.778 1.667H13" />
```
Comment (ericfennis) [inline icons/battery-charging.svg:12]: I can try this one as well in illustrator.

File: icons/contrast.svg (hunk 1 above)
Comment (jletey) [reply icons/contrast.svg:13]: Maybe 🤷‍♂️ I'll take a look now!

**Comment by jletey** [comment]
@ericfennis Yup, I'm currently in the process of refactoring all that code, so I used that script 😁

File: icons/battery-charging.svg (hunk 2 above)
Comment (ericfennis) [inline icons/battery-charging.svg:12]:
```suggestion
  <path d="M7 7H3.78A1.73 1.73 0 0 0 2 8.67v6.66A1.73 1.73 0 0 0 3.78 17H6M14 7h2.22A1.73 1.73 0 0 1 18 8.67v6.66A1.73 1.73 0 0 1 16.22 17H13" />
```

**Comment by jletey** [comment]
@ericfennis Thanks for that. I used your suggestion, and then ran `yarn build` again on it 😂

**Comment by ericfennis** [comment]
@jletey merge this one, or do you what to run it again?

**Comment by jletey** [comment]
Went ahead and merged this.

**Pull request merged**